hydrolib/core/dflowfm/extold/models.py

```diff
@@ -0,0 +1,417 @@
+from enum import Enum, IntEnum
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional, Union
+
+from pydantic import Field, root_validator, validator
+
+from hydrolib.core.basemodel import (
+    BaseModel,
+    DiskOnlyFileModel,
+    ModelSaveSettings,
+    ParsableFileModel,
+    SerializerConfig,
+)
+from hydrolib.core.dflowfm.extold.parser import Parser
+from hydrolib.core.dflowfm.extold.serializer import Serializer
+from hydrolib.core.dflowfm.polyfile.models import PolyFile
+
+
+class Quantity(str, Enum):
+    """Enum class containing the valid values for the boundary conditions category
+    of the external forcings.
+    """
+
+    # Boundary conditions
+    WaterLevelBnd = "waterlevelbnd"
+    """Water level"""
+    NeumannBnd = "neumannbnd"
+    """Water level gradient"""
+    RiemannBnd = "riemannbnd"
+    """Riemann invariant"""
+    OutflowBnd = "outflowbnd"
+    """Outflow"""
+    VelocityBnd = "velocitybnd"
+    """Velocity"""
+    DischargeBnd = "dischargebnd"
+    """Discharge"""
+    RiemannVelocityBnd = "riemann_velocitybnd"
+    """Riemann invariant velocity"""
+    SalinityBnd = "salinitybnd"
+    """Salinity"""
+    TemperatureBnd = "temperaturebnd"
+    """Temperature"""
+    SedimentBnd = "sedimentbnd"
+    """Suspended sediment"""
+    UXUYAdvectionVelocityBnd = "uxuyadvectionvelocitybnd"
+    """ux-uy advection velocity"""
+    NormalVelocityBnd = "normalvelocitybnd"
```

Comment:
Even though the `normalvelocitybnd, tangentialvelocitybnd` seems to be listed as one entry in the manual, I think you are correct that they should be two separate values 😃